vms/volunteer/utils.py

```diff
@@ -0,0 +1,19 @@
+from functools import wraps
+from django.shortcuts import render
+from django.http import Http404
+from volunteer.services import get_volunteer_by_id
+
+def vol_id_check(func):
+    @wraps(func)
+    def wrapped_view(request, volunteer_id):
+        vol = getattr(request.user, 'volunteer', hasattr(request.user, 'administrator'))
+        if not vol:
```

Comment:
@necessary129 I did not understand how this is possible. A user can be added through the admin interface by another admin. If a user is added to the system they are necessarily either an admin or a volunteer.